### PR TITLE
[Master] resume

### PR DESCRIPTION
**Pull request by aayushi93**

### CODE REVIEW

File: resume/gen_resume.sh [hunk 1]
```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
```
Comment (jarviscanada) [inline resume/gen_resume.sh:2]: missing readme file

**Review by jarviscanada — CHANGES_REQUESTED** [review]

File: resume/gen_resume.sh (hunk 1 above)
Comment (aayushi93) [reply resume/gen_resume.sh:2]: @jarviscanada Added. Please review.

File: resume/gen_resume.sh [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  init
+fi
+
+if [[ "$1" == *"_"* ]]; then
+  gen_resume $1
+fi
+
+echo "Error: invalid argument!"
+echo "Usage: ./$0 init|Firstname_Lastname"
+exit 1
```
Comment (jarviscanada) [inline resume/gen_resume.sh:54]: `resume/jarvis_resume_AAYUSHI_BHATNAGAR.pdf` wrong filename

File: README.md [hunk 3]
```diff
@@ -1,8 +1,62 @@
-# Jarvis Data Engineering Training
 
+# Aayushi Bhatnagar, Jarvis Consulting Group
```
Comment (jarviscanada) [inline README.md:2]: https://www.grammarly.com/
run a spelling and grammar check, please!!

**Review by jarviscanada — CHANGES_REQUESTED** [review]

File: resume/gen_resume.sh (hunk 2 above)
Comment (aayushi93) [reply resume/gen_resume.sh:54]: Fixed @jarviscanada

File: README.md (hunk 3 above)
Comment (aayushi93) [reply README.md:2]: Done and updated!

**Review by jarviscanada — APPROVED** [review]

**Pull request merged**